packages/contracts-bedrock/src/L2/SoulGasToken.sol

```diff
@@ -123,6 +123,8 @@ contract SoulGasToken is ERC20Upgradeable, OwnableUpgradeable {
     /// @notice batchMint is called:
     ///                        1. by EOA minters to mint SoulGasToken in batch when !IS_BACKED_BY_NATIVE.
     function batchMint(address[] calldata _accounts, uint256[] calldata _values) external {
+        // we don't explicitly check !IS_BACKED_BY_NATIVE here, because if IS_BACKED_BY_NATIVE,
+         // there's no way to add a minter.
```

Comment:
nit: extra space before //

Reply:
oh I re-did it by cherry-pick instead of manual copy in 0a9146ae17a574282553ece682ba899ae5871ca1.